doc/user/methods/gradient_boosting.md

```diff
@@ -0,0 +1,223 @@
+## `GradBoosting`
+
+The `GradBoosting` class implements a Gradient Boosting model that supports
+numerical and categorical features. The class offers several template 
+parameters and several runtime options that can be used to control the 
+model.
+
+Gradient Boosting is a very powerful ensemble algorithm used for both 
+classification and regression tasks. It utilizes a series of weak learners 
+(eg. [Decision Tree Regressor](decision_tree_regressor.md)) to arrive closer and closer to a targeted label. Each 
```

Comment:
You used `e.g.` here which means "for example" or "for instance", but in the implementation is hard-coded to use `DecisionTreeRegressor`.  So I would instead suggest:

```suggestion
(specifically [decision tree regressors](decision_tree_regressor.md)) to arrive closer and closer to a targeted label. Each 
```